ixmp/backend/base.py

```diff
@@ -882,15 +882,17 @@ def get_meta(self, s: Scenario):
         """
 
     @abstractmethod
-    def set_meta(self, s: Scenario, name, value):
-        """Set a single metadata key.
+    def set_meta(self, s: Scenario, name_or_data, value=None):
+        """Set single or multiple metadata entries.
 
         Parameters
         ----------
-        name : str
-            Metadata key name.
-        value : int or float or bool or str
-            Value for *name*.
+        name_or_data : str or dict
```

Comment:
suggest to call `name_or_dict`, because data is misleading (we are talking about meta indicators, not data in the traditional sense)

Reply:
updated